src/components/Table/TableHead/tools/sorting.js

```diff
@@ -0,0 +1,84 @@
+/**
```

Comment:
Do we need these files from Carbon?

Reply:
Now removed, after using const sortStates = {
NONE: 'NONE',
DESC: 'DESC',
ASC: 'ASC',
};